packages/foam-vscode/src/features/copy-without-brackets.spec.ts

```diff
@@ -1,21 +1,23 @@
-// import { env, window, Uri, Position, Selection, commands } from 'vscode';
-// import * as vscode from 'vscode';
+import { env, Position, Selection, commands } from 'vscode';
+import {
+  createFile,
+  getUriInWorkspace,
+  showInEditor,
+} from '../test/test-utils-vscode';
 
 describe('copyWithoutBrackets', () => {
   it('should pass CI', () => {
```

Comment:
We can remove this test now as it was only there because the `describe` requires a test case inside it